lang/en.json

```diff
@@ -111,16 +111,12 @@
   "WEAPON.BONUS": "Bonus",
   "WEAPON.DAMAGE": "Damage",
   "WEAPON.RANGE": "Range",
-  "WEAPON.FEATURE": "Feature",
-  "WEAPON.DESCRIPTION": "Description",
-  "WEAPON.DRAWBACK": "Drawback",
-  "WEAPON.APPEARANCE": "Appearance",
   "WEAPON.MELEE": "Melee",
   "WEAPON.RANGED": "Ranged",
   "WEAPON.1H": "1H",
   "WEAPON.2H": "2H",
-  "WEAPON.ARTIFACT_BONUS": "Artifact Bonus",
   "WEAPON.SKILL_BONUS": "Skill Bonus",
+  "WEAPON.FEATURE": "Feature",
 
   "ITEM.ATTACK": "Attack",
   "ITEM.BUILDING": "Building",
```

Comment:
Actually, there is something that could be avoided right here.
`ITEM.TypeThing` often duplicate `ITEM.THING`.

`ITEM.THING` is only used in the item chat posts, so it shouldn't be too hard to merge the two lists of strings.